app/models/messages/DepartureTransportMeans.scala

```diff
@@ -19,6 +19,7 @@ package models.messages
 import play.api.libs.json.{Json, OFormat}
 
 case class DepartureTransportMeans(
+  sequenceNumber: String,
```

Comment:
Is this being used? I appreciate it is part of the model in the DDNTA but if we don't use it then the reads won't be affected by omitting it from the case class

Reply:
Done